.nancy-ignore

```diff
@@ -1,3 +1,14 @@
-# Affects all versions of archiver which is required by vault.
-# Taken from: https://github.com/giantswarm/opsctl/pull/1072/files#diff-bbe4a7fb12c43622bce7c6840c770e9995be614626a219942ca138403629cb69R1
-CVE-2019-10743 until=2021-10-17
+# Hashicorp consul /api and /sdk are no longer intended for public use and don't have newer tags.
+# Waiting for direct dependencies to move away from them.
+CVE-2022-29153 until=2023-05-25
+CVE-2022-24687 until=2023-05-25
+
+# pkg:golang/github.com/kataras/iris/v12@v12.1.8
+CVE-2021-23772 until=2022-08-01
+
+# pkg:golang/github.com/urfave/negroni@v1.0.0
+sonatype-2021-1485 until=2022-08-01
+
+# pkg:golang/github.com/caddyserver/caddy@v1.0.3
+# Can't be updated without updating GS and K8s dependencies using newer logr version.
+CVE-2022-29718
```

Comment:
I added the same things [here](https://github.com/giantswarm/github/pull/583) so this doesn't get blown away by automation